app/services/invoices/payments/retry_service.rb

```diff
@@ -17,7 +23,7 @@ def call
           return result.not_allowed_failure!(code: 'payment_processor_is_currently_handling_payment')
         end
 
-        SendWebhookJob.perform_later(webhook_type, invoice) if customer&.organization&.webhook_url?
+        SendWebhookJob.perform_later(WEBHOOK_TYPE[invoice.invoice_type], invoice) if customer&.organization&.webhook_url?
```

Comment:
Layout/LineLength: Line is too long. [121/120] (https://rubystyle.guide#max-line-length)